src/coreclr/jit/lsra.cpp

```diff
@@ -459,15 +460,30 @@ RegRecord* LinearScan::getRegisterRecord(regNumber regNum)
 // Return Value:
 //     New regMask that has minRegCount registers after instersection.
 //     Otherwise returns regMaskActual.
-regMaskTP LinearScan::getConstrainedRegMask(regMaskTP regMaskActual, regMaskTP regMaskConstraint, unsigned minRegCount)
+regMaskTP LinearScan::getConstrainedRegMask(RefPosition* refPosition,
```

Comment:
Make it const?
```suggestion
regMaskTP LinearScan::getConstrainedRegMask(const RefPosition* refPosition,
```

Reply:
need to make `RegOptional()`, etc. `const` and then update its references...